dotcom-rendering/src/client/ophan/ophan.ts

```diff
@@ -87,7 +87,7 @@ export const getOphan = async (
 
 export const submitComponentEvent = async (
 	componentEvent: OphanComponentEvent,
-	renderingTarget: RenderingTarget,
+	renderingTarget: RenderingTarget = 'Web',
```

Comment:
this is currently being called by SDC components without a `renderingTarget` parameter

Reply:
Is this truly necessary? Could we update the consumers instead? With default parameters we make things more implicit…

Reply:
We can, I can make that change.
The `submitComponentEvent` gets passed into the components which currently live in SDC, but the type over there hasn't been updated to include the `renderingTarget` param.
Clearly the function works ok if you don't supply `renderingTarget`, but this highlights why we need to get away from this architecture!